frontend/src/components/accounts/__tests__/AccountRow.test.js

```diff
@@ -1,53 +0,0 @@
-import { renderI18nElement } from "../../../libraries/tester";
-import Enzyme, { shallow } from "enzyme";
-import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
-Enzyme.configure({ adapter: new Adapter() });
-import BN from "bn.js";
-
-import AccountRow from "../AccountRow";
-
-describe("<AccountRow />", () => {
-  beforeEach(() => jest.resetAllMocks());
-
-  it("should load amount", async () => {
-    const wrapper = shallow(
-      <AccountRow
-        accountId="account"
-        createdAt={Number(new Date(2019, 1, 1))}
-      />
-    );
-
-    const amount = new BN("100000000000000000000000000").add(
-      new BN("1000000000000000000000000")
-    );
-
-    wrapper.instance().state.amount = amount.toString();
-    wrapper.update();
-
-    expect(wrapper.instance().state.amount).toEqual(
```

Comment:
This test also didn't make much sense - in an essence it tested proper work of BN package